src/app/modules/time-entries/pages/time-entries.component.ts

```diff
@@ -125,7 +125,7 @@ export class TimeEntriesComponent implements OnInit, OnDestroy {
           uri: dataToUse.uri ? dataToUse.uri : '',
           activity_id: dataToUse.activity_id,
           project_id: dataToUse.project_id,
-          start_date: startDate,
+          start_date: new Date(),
           end_date: startDate
```

Comment:
I think you should change the name of the startDate constant to endDate for better readability.